luigi/contrib/redshift.py

```diff
@@ -37,6 +37,66 @@
                    "Will crash at runtime if postgres functionality is used.")
 
 
+class _MetadataColumnsMixin(object):
```

Comment:
This isn't distinct to Redshift as i believe this could also function in Postgres. Possibly move this mixin to `rdbms.py`?